src/main/java/sirius/biz/storage/layer2/BasicBlobStorageSpace.java

```diff
@@ -1616,9 +1616,8 @@ private void invokeConversionPipelineAsync(B blob, V variant) {
             eventRecorder.record(new BlobConversionEvent().withConversionProcess(conversionProcess)
                                                           .withConversionError(conversionException));
 
-            throw Exceptions.handle()
```

Comment:
shoudln't the ConversionEngine make this decision? Like throw a HandledException to only abort, but NPEs stil should end up in the syslog?